proto/tendermint/liquidity/v1beta1/liquidity.proto

```diff
@@ -265,14 +265,14 @@ message PoolBatch {
             format: "uint64"
         }];
 
-    // true if executed, false if not executed yet
+    // true if executed, false if not executed
     bool executed = 7 [(gogoproto.moretags) = "yaml:\"executed\"",
         (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
             example: "true",
         }];
 }
 
-// DepositMsgState defines the state of deposit message that contains state information as it is processed in the next batch(s)
+// DepositMsgState defines the state of deposit message that contains state information as it is processed in the next batch or batches.
```

Comment:
instead of using a parentheses for singular or plural, I recommend that we spell it out, per https://developers.google.com/style/plurals-parentheses